api/migrations/20210812174149-create-channels.js

```diff
@@ -24,6 +25,12 @@ module.exports = {
       memberCounter: {
         type: Sequelize.STRING,
       },
+      minecraft: {
```

Comment:
What does this column mean? The minecraft role, minecraft user name or something else?

Reply:
It is the minecraft channel in the Dev Launchers server. The name of the column isn't more specific because it is part of the channels model, and it is implied.